kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -17,35 +17,177 @@
     margin: 2em 4.5em;
 }
 
-#kb-data-staging-table_wrapper{
-    margin-top: 15px;
+
+/* BEGIN staging data table modifiers */
+
+.kb-staging-table {
+    table-layout: fixed;
+}
+
+.kb-staging-table-header .kb-staging-table-header__checkbox {
```

Comment:
is the `.kb-staging-table-header` needed? (and similarly below)

Reply:
good catch - not needed in all places, will fix where i can